sr/comp/comp.py

```diff
@@ -67,10 +108,12 @@ def __init__(self, root: str | Path) -> None:
         self.num_teams_per_arena = len(self.corners)
 
         scorer = load_scorer(self.root)
+        ranker = load_ranker(self.root)
         self.scores = scores.Scores.load(
             self.root,
             self.teams.keys(),
             scorer,
+            ranker,
             self.num_teams_per_arena,
```

Comment:
We're getting to an argument count where this might be better expressed with kwargs.

Reply:
Hrm, perhaps. Each of these would be of the form `foo=foo` though. As we're fully type checked too we've got some protection against errors. I'm not going to change this now, but will bear it in mind if this gets any bigger.